site/source/user-manual/service-guides/nextcloud/nextcloud-apps/joplin.rst

```diff
@@ -0,0 +1,105 @@
+.. _nc-joplin:
+
+==================
+Joplin Integration
+==================
+`Joplin <https://joplinapp.org/>`_  is an open source note-taking app. Capture your thoughts and securely access them from any device.  First set up your :ref:`client devices<nextcloud-setup>`, so that Nextcloud will automatically keep your notes synced.
+
+Clients available for:
+
+- Desktop
+    - Linux
+    - Mac
+    - Windows
+
+- Mobile
+    - Android
+    - iOS
```

Comment:
Clients are also listed in the link on line 6; is it necessary to list them here as well?

Reply:
Probably not, but this was the convention across the other Nextcloud apps in this section.  Kept it for uniformity